components/button/test/button-add.test.js

```diff
@@ -0,0 +1,24 @@
+import '../button-add.js';
+import { fixture, html, oneEvent, runConstructor } from '@brightspace-ui/testing';
+
+describe('d2l-button-add', () => {
+
+	describe('constructor', () => {
+
+		it('should construct', () => {
+			runConstructor('d2l-button-add');
+		});
+
+	});
+
+	describe('events', () => {
+
+		it('dispatches d2l-button-add-click event when clicked', async() => {
+			const el = await fixture(html`<d2l-button-add></d2l-button-add>`);
+			setTimeout(() => el.click());
```

Comment:
Same here for the mouse.

```suggestion
			setTimeout(() => clickElem(el));
```